client/interfaces.go

```diff
@@ -426,6 +426,9 @@ type InstanceServer interface {
 	RawQuery(method string, path string, data any, queryETag string) (resp *api.Response, ETag string, err error)
 	RawWebsocket(path string) (conn *websocket.Conn, err error)
 	RawOperation(method string, path string, data any, queryETag string) (op Operation, ETag string, err error)
+
+	// Documentation
+	GetLxdDocumentationConfig() (string, error)
```

Comment:
Why does this need to be added to the client?

Reply:
I thought it'd be useful for the LXD UI folk to have a client function to easily get the documentation config

Reply:
The ui is written in JavaScript so they don't use the go client

Reply:
Oh ok then I don't think this new function is truly necessary. I can remove it.

Reply:
Thanks yes please